runtime/modules/java/module/src/main/java/io/atlasmap/java/module/JavaValidationService.java

```diff
@@ -201,7 +199,7 @@ private void validateClass(String mappingId, JavaField field, List<Validation> v
         }
     }
 
-    protected Integer detectClassVersion(String className) {
+    Integer detectClassVersion(String className) {
```

Comment:
should it be package private? for what purpose? It just looks like a private routine though.

Reply:
out this class this method  is referenced only in the testcases. Shall I continue using the protected

Reply:
Then this should be private and test should be modified to run from public methods... that's the primary goal for this issues...

Reply:
Then I think,in this case we can remove the testcase